I have some troubles to build

I want to make some adjustments / PRs but I have some bigger problems to get it running. I don't know if it is a problem with the plugin or with my environment.

All issues are happening with a clean copy of the branch `develop`

## Issues:
### ~~1. `install`~~
#### `npm install`:
```
npm ERR! While resolving: webpack-localize-assets-plugin@0.0.0-semantic-release
npm ERR! Found: webpack@4.42.0
npm ERR! node_modules/webpack
npm ERR!   dev webpack@"4.42.0" from the root project

Could not resolve dependency:
npm ERR! peer webpack@"^4.44.2" from webpack-manifest-plugin@3.2.0
```
#### `pnpm install`
```
 ERR_PNPM_PEER_DEP_ISSUES  Unmet peer dependencies
.
└─┬ webpack-manifest-plugin 3.1.1
  └── ✕ unmet peer webpack@>=4.44.2: found 4.42.0
```

Changing to `4.44.2` solves the problem:
https://github.com/privatenumber/webpack-localize-assets-plugin/blob/77a5c924b5b5e6794ef9f38182722d8510549222/package.json#L71

### ~~2. `pnpm run build`:~~
`dist` is generated but following is printed. I installed the packages with `pnpm install`. With `npm install` the error is not showing.
```
> webpack-localize-assets-plugin@0.0.0-semantic-release build D:\Daten\Projekte\Github\webpack-localize-assets-plugin
> tsc

node_modules/.pnpm/@types+mini-css-extract-plugin@2.4.0/node_modules/@types/mini-css-extract-plugin/index.d.ts:10:23 - error TS4090: Conflicting definitions for 'node' found at 'D:/Daten/Projekte/Github/webpack-localize-assets-plugin/node_modules/.pnpm/@types+node@16.11.6/node_modules/@types/node/index.d.ts' and 'D:/Daten/Projekte/Github/webpack-localize-assets-plugin/node_modules/.pnpm/@types+node@15.0.2/node_modules/@types/node/index.d.ts'. Consider installing a specific version of this library to resolve the conflict.
  The file is in the program because:
    Type library referenced via 'node' from file 'D:/Daten/Projekte/Github/webpack-localize-assets-plugin/node_modules/.pnpm/@types+webpack@4.41.32/node_modules/@types/webpack/index.d.ts' with packageId 
'@types/node/index.d.ts@15.0.2'
    Type library referenced via 'node' from file 'D:/Daten/Projekte/Github/webpack-localize-assets-plugin/node_modules/.pnpm/@types+mini-css-extract-plugin@2.4.0/node_modules/@types/mini-css-extract-plugin/index.d.ts' with packageId '@types/node/index.d.ts@16.11.6'

10 /// <reference types="node" />
                         ~~~~

  node_modules/.pnpm/@types+webpack@4.41.32/node_modules/@types/webpack/index.d.ts:28:23
    28 /// <reference types="node" />
                             ~~~~
    File is included via type library reference here.

node_modules/.pnpm/@types+webpack-sources@2.1.0/node_modules/@types/webpack-sources/index.d.ts:7:23 - error TS4090: Conflicting definitions for 'node' found at 'D:/Daten/Projekte/Github/webpack-localize-assets-plugin/node_modules/.pnpm/@types+node@16.11.10/node_modules/@types/node/index.d.ts' and 'D:/Daten/Projekte/Github/webpack-localize-assets-plugin/node_modules/.pnpm/@types+node@15.0.2/node_modules/@types/node/index.d.ts'. Consider installing a specific version of this library to resolve the conflict.
  The file is in the program because:
    Type library referenced via 'node' from file 'D:/Daten/Projekte/Github/webpack-localize-assets-plugin/node_modules/.pnpm/@types+webpack@4.41.32/node_modules/@types/webpack/index.d.ts' with packageId 
'@types/node/index.d.ts@15.0.2'
    Type library referenced via 'node' from file 'D:/Daten/Projekte/Github/webpack-localize-assets-plugin/node_modules/.pnpm/@types+webpack-sources@2.1.0/node_modules/@types/webpack-sources/index.d.ts' with packageId '@types/node/index.d.ts@16.11.10'

7 /// <reference types="node" />
                        ~~~~

  node_modules/.pnpm/@types+webpack@4.41.32/node_modules/@types/webpack/index.d.ts:28:23
    28 /// <reference types="node" />
                             ~~~~
    File is included via type library reference here.

src/multi-locale.ts:2:64 - error TS7016: Could not find a declaration file for module 'webpack-sources'. 'D:/Daten/Projekte/Github/webpack-localize-assets-plugin/node_modules/.pnpm/webpack-sources@2.2.0/node_modules/webpack-sources/lib/index.js' implicitly has an 'any' type.
  If the 'webpack-sources' package actually exposes this module, consider sending a pull request to amend 'https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/webpack-sources'

2 import { RawSource, SourceMapSource, SourceAndMapResult } from 'webpack-sources';
                                                                 ~~~~~~~~~~~~~~~~~


Found 3 errors.

 ELIFECYCLE  Command failed with exit code 2.
```

### 3. Tests are failing
1. `npm run build`
2. `npm run test`
```
 Webpack 4.44.2 › passing › dynamically load relative locale json path

    expect(received).toBe(expected) // Object.is equality

    Expected: "Hello World"
    Received: "Hello world-key"

      942 |                     delete watching.require.cache['/dist/index.en.js'];
      943 |                     enBuild = watching.require('/dist/index.en.js');
    > 944 |                     expect(enBuild).toBe('Hello World');
          |                                     ^
      945 |
      946 |                     watching.fs.writeFileSync('/src/locales/en.json', JSON.stringify({
      947 |                             'world-key': 'World',

      at Object.<anonymous> (tests/webpack-localize-assets-plugin.spec.ts:944:20)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 2 skipped, 40 passed, 43 total
```

### ~~4. `npm run test:wp5` - `webpack-cli`~~
```
One CLI for webpack must be installed. These are recommended choices, delivered as separate packages:
 - webpack-cli (https://github.com/webpack/webpack-cli)
   The original webpack full-featured CLI.
We will use "npm" to install the CLI via "npm install -D".
Do you want to install 'webpack-cli'
```
I suggest to add `webpack-cli` to the dependencies

### 5. `npm run test:wp5` - not executing
```
> webpack-localize-assets-plugin@0.0.0-semantic-release test:wp5
> WEBPACK=5 jest

[webpack-cli] Unknown command or entry '=5'
[webpack-cli] Run 'webpack --help' to see available commands and options
```

### 6. `npm install ../webpack-localize-assets-plugin`
I want to include the local build (`develop`) of `webpack-localize-assets-plugin` in a other project but when I run `npm install ../webpack-localize-assets-plugin` and then build my project I get following error. (it's working fine with the npm-server version)
```
ERROR in ./MY_PPROJECT/index.js
No template for dependency: ConstDependency
```

## Environment

- webpack-localize-asets-plugin version: `develop`
- Webpack version: `5.69.1`
- Operating System: Windows 10
- Node version: v16.13.1
- Package manager (npm/yarn/pnpm) and version: npm 7.24.1

--- a/tests/webpack-localize-assets-plugin.spec.ts
+++ b/tests/webpack-localize-assets-plugin.spec.ts
@@ -939,7 +939,7 @@ describe(`Webpack ${webpack.version}`, () => {
 
 			expect(stats.hasWarnings()).toBe(false);
 
-			delete watching.require.cache['/dist/index.en.js'];
+			delete watching.require.cache[watching.require.resolve('/dist/index.en.js')];
 			enBuild = watching.require('/dist/index.en.js');
 			expect(enBuild).toBe('Hello World');
 
@@ -953,7 +953,7 @@ describe(`Webpack ${webpack.version}`, () => {
 			expect(warnings.length).toBe(1);
 			expect(warnings[0].message).toMatch('Missing localization for key "hello-key" used in /src/index.js:1:15 from locales: en');
 
-			delete watching.require.cache['/dist/index.en.js'];
+			delete watching.require.cache[watching.require.resolve('/dist/index.en.js')];
 			enBuild = watching.require('/dist/index.en.js');
 			expect(enBuild).toBe('hello-key World');
 

